news/2025/2025-10-05-vocaloid-mapping-contest-3.md

```diff
@@ -0,0 +1,90 @@
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+---
+
+**mimizu** is the second new Featured Artist. As one of the forefront artists behind the Kasane Teto revolution of 2025, **mimizu** creates fun, upbeat songs using Kasane Teto, our favorite VOCALOID gremlin. Known for incredibly catchy and fun instrumental arrangements, **mimizu** is no stranger to rhythm gaming. Heck, they have a whole song themed around taiko: [Tatakina](https://www.youtube.com/watch?v=DpHt9ys-6I8&feature=youtu.be)!
+
+**5** new songs are a part of [**mimizu**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/505)!
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+The songs are ready. The stage is set. ***Here's what's coming next.***
+
+## Vocaloid Mapping Contest 3: SWAP//DUET
+
+The [**Vocaloid Mapping Contest** (***VMC***)](/wiki/Contests/VMC) is back! 
+
+The last contest expanded the scope from osu! to including every game mode, as well as the Vocaloid Appreciation Team's first Featured Artist addition of iroha(sasaki). Since it's been a while since the last event, you may wish to refresh yourself on what the outcome of the last VMC was [here](/wiki/Contests/VMC/2)!
+
+This time, the spotlight shines on **two Featured Artists**: **MARETU** and **mimizu**. Teams of two will work together across three unique phases — **PARALLEL**, **SWAP**, and **DUET** — to create the best VOCALOID maps possible.
+
+- **PARALLEL:** Each teammate maps a different artist's song.  
+- **SWAP:** Partners swap songs and remap each other's picks with a completely new interpretation.  
+- **DUET:** Partners must collaborate directly to create a single cohesive mapset.  
+
+Full rules, judging criteria, and song downloads are available on the [contest wiki page](/wiki/Contests/VMC/3) and the [contest website](https://www.vat.gg/vmc3).
+
+---
+
+### Submission schedule
+
+| Event | Date |
+| --: | :-- |
+| PARALLEL Submission Deadline | November 9, 2025 (23:59 UTC) |
+| Screening | November 10 - November 15, 2025 |
+| SWAP Submission Deadline | December 7, 2025 (23:59 UTC) |
+| SWAP Results Stream | December 21, 2025 (18:00 UTC) |
+| DUET Submission Deadline | February 22, 2026 (23:59 UTC) |
+| DUET Results Stream | March 15, 2026 (23:59 UTC) |
+
+Results will be streamed live with commentary on the [*Vocaloid Appreciation Team* Twitch channel](https://www.twitch.tv/vocaloidappreciationteam), and posted afterwards on YouTube.
+
+---
+
+### Ruleset
+
+- **Submissions must use songs from [**MARETU**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/504) and  [**mimizu**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/505).** Cut versions are not allowed.  
```

Comment:
maybe emphasise this since it's unlike mappers guild quests or mbc which run more often?

```suggestion
- **Submissions must use songs from [**MARETU**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/504) and  [**mimizu**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/505).** Cut versions are *not* allowed.  
```